src/components/cell-control/autocompleter.test.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { createRef } from 'react';
+import { render as _render } from '@testing-library/react';
+import Autocompleter from './autocompleter.jsx';
+
+const render = (ui, options = { labelledBy: 'labelled-by' }) => (
+  _render(ui, {
+    ...options,
+    wrapper: props => (
+      <table>
+        <thead>
+          <tr>
+            <th id={options.labelledBy}>Column Header</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            {props.children}
+          </tr>
+        </tbody>
+      </table>
+    ),
+  })
+);
+
+describe('Autocompleter cell control', () => {
+  const requiredProps = {
+    id: 'test-id',
+    labelledBy: 'labelled-by',
+    name: 'test-name',
+  };
+
+  it('has defaults', () => {
+    const ref = createRef();
+    render(<Autocompleter {...requiredProps} ref={ref} />);
+    expect(document.body).toMatchSnapshot();
+    expect(document.body).toBe(document.activeElement);
+    expect(ref.current).toMatchSnapshot();
+  });
+
+  describe('className API', () => {
```

Comment:
Do we want a test for the `ref` api? 

And also I know this is a conversation we've beaten to death, but how much of the underlying `autocompleter` do we want to test? Have we decided that since the control itself is tested we're okay with skimping on the cell vs form control version?

Reply:
Yeah, I've got another story to revisit the test coverage. For now, I am definitely adding test coverage on the "broken bits" as I integrate with bigmaven. I'm hoping it offers some glimpse at what is reasonable test coverage.

For now, I am just using the fact that the default test has an assertion against the ref API already -- I don't think we need more than that. If I had that originally, I think I wouldn't have had to do this bugfix.